src/main/java/nextstep/ladder2/player/Players.java

```diff
@@ -0,0 +1,73 @@
+package nextstep.ladder2.player;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Players {
+    private List<Player> players = new ArrayList<>();
+    private static final Player NONE = Player.of("NONE", Integer.MAX_VALUE-1, Integer.MAX_VALUE);
+    public Players(List<Player> players) {
+        this.players = players;
+    }
+
+    private Players(String... names) {
+        this.players = makeGroup(names);
+    }
+
+    public static Players of(String s) {
+        String[] names = s.split(",");
+        return new Players(names);
+    }
+
+    private List<Player> makeGroup(String[] names) {
```

Comment:
메소드 순서도 코드 흐름에 맞게 재구성 해주면 어떨까요?
```suggestion
    private Players(List<Player> players) {
        this.players = players;
    }

    public static Players of(List<Player> players)) {
        return new Players(players);
    }

    public static Players of(String s) {
        return of(makeGroup(s.split(",")));
    }

    private List<Player> makeGroup(String[] names) {
```
그리고 makeGroup도 아예 of로 표현해주면 더 좋지 않을까 싶네요 ㅎㅎ